cloudslang-cli/src/main/resources/log4j.properties

```diff
@@ -4,7 +4,7 @@ def.files.backup.count=10
 log4j.rootLogger=INFO, A1
 
 log4j.appender.A1=org.apache.log4j.RollingFileAppender
-log4j.appender.A1.File=logs/execution.log
+log4j.appender.A1.File=${user.dir}/target/cslang/logs/execution.log
```

Comment:
seems to me that `${user.dir}` returns the same as `.` which does not solve [this issue](https://github.com/CloudSlang/cloud-slang/pull/482#issuecomment-157315259)

please investigate further